plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildInputs/MemberType.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.UniversalReference
+{
+    using System;
+
+    [Serializable]
+    public enum MemberType
```

Comment:
change to string

---
In reply to: [112601880](https://github.com/dotnet/docfx/pull/1574#discussion_r112601880) [](ancestors = 112601880)